docs/Modules/index.md

```diff
@@ -0,0 +1,83 @@
+# Modules
+
+## Base Classes
+
+Depending on which `ScriptModule` you desire the module to be in, you change the inheritance to allow for more events. All the possible events in the modules are documented within the the respective classes.
+
+| Script Module | CF Module        |
+| ------------- | ---------------- |
+| 1_Core        | `CF_ModuleCore`  |
+| 2_GameScript  | `CF_ModuleCore`  |
+| 3_Game        | `CF_ModuleGame`  |
+| 4_World       | `CF_ModuleWorld` |
+| 5_Mission     | `CF_ModuleWorld` |
+
+## Events
+
+All events within the modules are by default disabled. Once enabled the event can't be disabled. To enable an event you call the corresponding method starting with 'Enable'. 
+
+| Event Method                              | Enable Method                 | When Fired?                                        | Executing Platform |
+| ----------------------------------------- | ----------------------------- | -------------------------------------------------- | ------------------ |
+| `CF_ModuleCore::OnMissionStart`           | `EnableMissionStart`          | Called when `MissionBase::OnMissionStart` is ran   | ALL                |
+| `CF_ModuleCore::OnMissionFinish`          | `EnableMissionFinish`         | Called when `MissionBase::OnMissionFinish` is ran  | ALL                |
+| `CF_ModuleCore::OnMissionLoaded`          | `EnableMissionLoaded`         | Called when `MissionBase::OnMissionLoaded` is ran  | ALL                |
```

Comment:
Is it really "is ran"? I thought it was "is run" or "was ran" but not a mix of it possible. Not sure if that is British vs American English or something.